### ISSUE TITLE
 invalid stored block lengths

### ISSUE THREAD

**Issue by Grey2k** [avocode/avocode#625]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.5.2
Source UID: 86482
```
Error: invalid stored block lengths
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by vojtatranta** [comment]
Hi, thanks for your report.
This is duplicated issue. More info is to be found here: #607

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
